ru/utils.py

```diff
@@ -743,7 +744,9 @@ def get_barcode_kits(address, timeout=10000):
     return res
 
 
-def get_barcoded_run_info(toml_filepath, num_channels=512, validate=True):
+def get_barcoded_run_info(
```

Comment:
Are the changes to this function needed?

Reply:
Which changes

Reply:
All of them, if we're picking up the param from the CLI we can pass it through to `simple_analysis` without changing the TOML or it's parsing at all.

Reply:
I didn't want to change `simple_analysis` in case it was standardised between scripts - but if it doesn't matter I'll pass it through
It's easier that way to keep it when re reading the TOML as well

Reply:
😅 `get*_run_info` is much more precarious